asf_search/ASFSearchOptions/ASFSearchOptions.py

```diff
@@ -69,7 +69,7 @@ def __str__(self):
         """
         What to display if `print(opts)` is called.
         """
-        return json.dumps(dict(self), indent=4)
+        return json.dumps(dict((key, value) for key, value in self if key != 'session'), indent=4)
```

Comment:
What's the point of this? And would it be useful to skip ALL the keys in config? https://github.com/asfadmin/Discovery-asf_search/blob/master/asf_search/ASFSearchOptions/config.py

Reply:
`requests.session.Session` isn't serializable, so json.dumps() raises a value error (since search options __iter__ ignores config defaults this only happens when ASFSession is the non-default one). The other config options are better left in since they provide information about the query asf-search ran.
Making `ASFSession` json serializable would be a decent feature for a future pr given it now has constructor params

Reply:
Ah makes sense. I actually hit that problem over the weekend with datetime objects being in lambda context I was trying to log. This worked for me:

```python
print(json.dumps({"Event": event, "Context": context}, default=str))
```

What do you think about adding `default=str` instead, if this works here too? It just feels weird only skipping a very specific item. And if we add more items later on, they won't randomly break here. It might be weird seeing an object you can't do anything with, but at least this way they also see the key and know it's apart of the SearchOptions object too

Reply:
I didn't know about the `default` keyword, just added that in the original line, thank you!